apps/runtime-demo/3006-runtime-remote/webpack.config.js

```diff
@@ -0,0 +1,89 @@
+const { registerPluginTSTranspiler } = require('nx/src/utils/nx-plugin.js');
+registerPluginTSTranspiler();
+
+const { composePlugins, withNx } = require('@nx/webpack');
+const { withReact } = require('@nx/react');
+
+const path = require('path');
+const baseConfig = require('./module-federation.config');
+// const { withModuleFederation } = require('@nx/react/module-federation');
+const { ModuleFederationPlugin } = require('@module-federation/enhanced');
+
+/**
+ * @type {import('@nx/react/module-federation').ModuleFederationConfig}
+ **/
+const defaultConfig = {
+  ...baseConfig,
+};
+
+module.exports = composePlugins(
+  withNx(),
+  withReact(),
+  async (config, context) => {
+    config.plugins.push(
+      new ModuleFederationPlugin({
+        name: 'runtime_remote',
+        // library: { type: 'var', name: 'runtime_remote' },
+        filename: 'remoteEntry.js',
+        exposes: {
+          './Button': './src/Button.tsx',
+          './Button1': './src/Button1.tsx',
+        },
+      }),
+    );
+    console.log(
+      'config.optimization?.runtimeChunk',
+      config.optimization?.runtimeChunk,
+    );
+    config.optimization.runtimeChunk = false;
```

Comment:
we should include one of these apps at some point with single runtime chunk true, to make sure everything works with both build setups. I think there is some additional work needed to support single runtimr chunk automatically though. but in future :P